README.md

```diff
@@ -13,7 +13,9 @@ Repository to describe, develop, document and test the QualityOnDemand API famil
 
 * Service APIs for “Quality on Demand” (see APIBacklog.md)
 * It provides the customer with the ability to:  
-  * set quality for access network connections (e.g. mobile device connection or fixed access between a home gateway and the service providers gateway router)  
+  * set quality for access network connections (e.g. mobile device connection or fixed access between a home gateway and the service providers gateway router)
+      * Session mode, for a specific duration
+      * Provision mode, indefinitely for each time that the device connects to the network
```

Comment:
```suggestion
      * Provision mode, indefinitely for each time the device connects to the network
```

Question: Should we specific that this is just when it connects to the current network or any network.  Saying "the network" is a little ambiguous, would this apply when roaming or not?